frontend/src/pages/secret-manager/OverviewPage/OverviewPage.tsx

```
@@ -845,6 +863,22 @@ export const OverviewPage = () => {
                   </DropdownMenuTrigger>
                   <DropdownMenuContent align="end">
                     <div className="flex flex-col space-y-1 p-1.5">
+                      <Button
+                        leftIcon={<FontAwesomeIcon icon={faFingerprint} className="pr-2" />}
+                        onClick={() => {
+                          if (subscription && subscription.dynamicSecret) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)